packages/pyright-internal/src/analyzer/service.ts

```diff
@@ -645,36 +645,6 @@ export class AnalyzerService {
                     }
                 }
             }
-        } else {
-            const importFailureInfo: string[] = [];
```

Comment:
This block is only for logging, and has been moved down.

Reply:
IIRC, we previously cached the results of the call to Python, so this code path was effectively "free". But it appears that we no longer cache this value (at least not on this code path), so it results in a redundancy. I agree that it's fine to remove this.